src/solver/simulation/sim_structure_probleme_economique.h

```diff
@@ -433,6 +437,9 @@ typedef struct
     double* ValeursHorairesDeDefaillanceNegativeAny;
 
     double* ValeursHorairesDeDefaillanceEnReserve;
+
+    double* HourlyValuesDomesticEnergyNotServed;
```

Comment:
This is for economy mode. anything we need to do for adequacy and draft mode?

Reply:
I checked with Jean-Marc from RTE, it is only expected for us to make this feature work in economy mode.